app/usecases/clans.py

```diff
@@ -23,25 +38,7 @@ async def update_clan(user_id: int) -> str:
     )
 
     if not clan_tag:
-        CLANS[user_id] = ""
-        return ""
+        clan_tag = ""
 
-    CLANS[user_id] = clan_tag
+    await cache(user_id, clan_tag)
     return clan_tag
-
-
-async def load_clans() -> None:
```

Comment:
why did we remove this? this will literally mean clan tags will only update once every 24 hours if not for this

Reply:
Now about these, I feel a cron job would be much more suited, with some cache management in RealistikPanel/Hanayo/API/All three